tests/integration/components/lt-body-test.js

```diff
@@ -33,7 +33,7 @@ test('row selection - enable or disable', function(assert) {
 
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions canSelect=canSelect}}`);
 
-  let row = this.$('tr:first');
+  let row = this.$('tr:not(.lt-scaffolding-row):first');
```

Comment:
Got it. That makes sense.

It's a little confusing since it isn't set.

As an aside there aren't any actual tests that verify `enableScaffoliding` puts a scaffolding row in the DOM :astonished:  :rotating_light: :roll_eyes: